### ISSUE TITLE
[BUG] Emoji-picker

### ISSUE THREAD

**Issue by SMAW** [CorentinTh/it-tools#555]
**Which tool is impacted?**
Emoji-picker

**To Reproduce**
When clicking an emoji to text it should copy it to clipboard (it shows a message for it) but it doesn't seem to work in Windows 10 Edge/Firefox/Chrome. When pasting it somewhere it paste the thing I had on clipboard. Manual ctrl+c / ctrl+v for copying works

**Expected behavior**
It copies the emoji/text to my clipboard.

**Screenshots**

**Configuration (please complete the following information):**

- Device: Desktop computer
- OS: Windows 10
- Browser Edge/Chrome/Firefox
- Version all latest (browsers)

**Additional context**
Thanks for adding the requested feature, really like it!... You even made it better to automatically copy it to the clipboard, but it doesn't work (for me)

**Comment by CorentinTh** [comment]
Hello @SMAW !
Thank you for this quick bug report, it's now fixed 👌🏻

**Comment by PDXUser** [comment]
I have the same issue in Linux  on my Desktop system  using Firefox and Vivaldi and Chromium/Chrome browser for testing.   Installed via Docker.

**Comment by CorentinTh** [comment]
Hi @PDXUser 
The issue has been fixed and a new it-tools docker version has been released, please update your instance to fix the issue

**Comment by PDXUser** [comment]
Hi @CorentinTh ! 

Thanks for your reply, it happens that I am using the latest at least as from what Watchtower told me.  In recalling the issue tonight, I was reminded that overall, copying from the interface at does not work at all period.... So I did some research in the closed bugs and found #342 which is exactly the same thing I am experiencing.    I always access it via the LAN IP which therein  is my issue - I had to proxy the app behind my primary domain so it's technically exposed to the cloud (I don't mind so much about that) and with SSL enabled, the copy works as expected.   And as proof: 🎈  straight from the Emoji selector.
